content/methods/date-adjustment.jsonnet

```diff
@@ -13,10 +13,14 @@
   },
   specification_link: null,
   code_link: 'https://github.com/ONSdigital/sml-python-small/blob/main/sml_small/date_adjustment.py',
+  readme_link: 'https://github.com/ONSdigital/sml-python-small/blob/main/docs/date_adjustment.md',
```

Comment:
I though we were going to call it user documentation link rather than readme ? (we had a convo over Teams if this is the change I think it was?)

Reply:
Yes in the frontend its been changed but not the backend :) but i can change this as well, the reason why I left it is becuase readme link makes more sense to dev's. Happy to change it now :)